content/en/Getting started/glossary.md

```diff
@@ -75,6 +75,65 @@ refines best practice security solutions.
 One of the test criteria used by our pentesters is [CIS Controls v8](https://www.cisecurity.org/controls/v8),
 released in 2021.
 
+## Cobalt Users
+
+When using the Cobalt UI, you may encounter a variety of different users, in the following
+categories:
+
+- Organization Roles: If you're a Cobalt customer, your account may have one or more of the following roles:
+  - Organization Owner: Also known as an _Org Owner_
+  - Organization Member: Also known as an _Org Member_
+  - Pentest Team Member: Also known as a _PT Team Member_
+
+- Pentester Roles: If you're a Cobalt pentester, you may be a _PT Pentester_ or a _PT Lead_.
+
+Select Cobalt employees may be assigned as administrators, as _Cobalt Staff_.
+
+You can review a list of permissions associated with each organization role in the following
+article: [What do the user roles mean?](https://cobaltio.zendesk.com/hc/en-us/articles/360057093472-What-do-the-user-roles-mean-).
+
+<!-- Per https://zombie.atlassian.net/browse/DOCS-5 I should add info from an internal
+spreadsheet, but need to find pull code in Hugo -->
+
+### Organization Member
+
+An _Org Member_ is a user of the Cobalt App who can create an [asset](#asset) or a
+[pentest](#pentest). That user can also see the pentesters who are working on their asset.
+If allowed by their {{% ptaas-tier %}}, they can also manage integration with Jira and GitHub.
+
+### Organization Owner
+
+An _Org Owner_ is the administrator for a customer organization within the Cobalt app. As such,
+they can:
+
+- Add/remove the users of their choice, by their email addresses, as an _Org Member_ or a
+  _PT Team Member_.
+- View collaborators, the members of their pentest team. That team includes 
+  - _PT Team Members_
+  - _PT Lead_
+  - _PT Pentester_ (when applicable)
+- If allowed by their {{% ptaas-tier %}}, they can also manage
+[SAML](#security-assertion-markup-language) settings for the users in their organization.
+
+### Pentest Team Member
+
+A _PT Team Member_ is a customer representative during a pentest. That user can review and
+respond to each [finding](#finding) identified by a Cobalt pentester.
```

Comment:
There is a whole finding state workflow which I don't know if it is in the scope of this documentation. There was even a hackathon project reflecting on it [HACK-62](https://zombie.atlassian.net/browse/HACK-62)

Reply:
I'll review HACK-62 before I figure out what to do here.